pkg/controller/certrotation/helpers.go

```diff
@@ -0,0 +1,38 @@
+package certrotationcontroller
+
+import (
+	"encoding/json"
+
+	ctrlcommon "github.com/openshift/machine-config-operator/pkg/controller/common"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/klog/v2"
+)
+
+func isUserDataSecret(secret corev1.Secret) bool {
```

Comment:
I think this function is fine, but I am curious what we should consider our boundary of responsibilities, since as I understand it we'd update any secret that looks like a user-data?. I guess a user generally wouldn't be providing their own certificates, so this really shouldn't ever overwrite any user-provided content?

Reply:
Currently we error out if there is more than one CA in here, but as part changing the source of the secret to the configmap, we'll have to make it support multiple CAs. I'll make it a bit smarter to ensure any other ones are left untouched (: